.github/workflows/update_v8_snapshot_cache.yml

```diff
@@ -1,10 +1,136 @@
 name: Update V8 Snapshot Cache
-on: [workflow_dispatch]
+on:
+  schedule:
+    # Run every Wednesday at 00:00 UTC
+    - cron: '0 0 * * 3'
+  push:
+    branches:
+      - ryanm/feature/v8-snapshots-auto-pr
```

Comment:
Remove before merging?

Reply:
I could. But I also sort of see this as similar to what we do in the circle ci file. If you're working on it on your branch, update it with your branch, and so on and so on.